chapters/101.rst

```diff
@@ -159,6 +181,39 @@ Driver
 	:width: 25%
 
 A Driver “translates” input and output messages for an Executor. Each Executor requires a different data format to perform its task. Therefore, a Driver interprets incoming messages into Documents and extracts required fields for an Executor. 
+It is thus a translation layer between Jina data type and Python or Numpy data type. It makes the Executor agnostic to data type and network. Without the driver, you will need to convert, pass, extract back and forth between the general data type and the Numpy data type.
+
+You can define the Executor's behavior under the network request using drivers. It translates to -
+while receiving request x, call the Executor and do y.
+Your algorithm can hence directly handle the request - that's exactly the purpose of the driver.
+As an algorithm developer, driver is invisible to you and you don't have to worry about it.
+In general, the Executor and driver work together and serve as a logic layer in Jina.
+
+
+
+App
+=============
+
+Jina as a framework supports abstractions at different layers and exposes them as an API to the users or developers. Some high-level APIs are public some intermediate or low-level apis
+are highlighted intentionally so developers can stay focused.
```

Comment:
Can you check it's always API not api ?

Reply:
The sentence starting "Some high-level APIs..." doesn't make sense to me?

Reply:
Sure, done. I've made it more precise